module/input/GameController/src/GameController.cpp

```diff
@@ -86,6 +88,16 @@ namespace module::input {
                           std::tie(listenHandle, std::ignore, std::ignore) =
                               on<UDP::Broadcast, With<GameState>>(recieve_port)
                                   .then([this](const UDP::Packet& p, const GameState& gameState) {
+                                      std::string remoteAddr = ipAddressIntToString(p.remote.address);
+
+                                      if (!udp_filter_address.empty() && remoteAddr != udp_filter_address) {
+                                          log<NUClear::INFO>("Ignoring UDP packet from",
+                                                             remoteAddr,
+                                                             "as it doesn't match configured filter address",
+                                                             udp_filter_address);
+                                          return;
+                                      }
```

Comment:
It is worth doing something like this, don't really want to know it's filtering every packet.. kinda annoying spam
```
static bool hasPrinted = false;
if (!udp_filter_address.empty() && remoteAddr != udp_filter_address) {

    if (!hasPrinted) {
        log<NUClear::INFO>("Ignoring UDP packet from",
                            remoteAddr,
                            "as it doesn't match configured filter address",
                            udp_filter_address);
        hasPrinted = true;
    }
    return;
}```

Reply:
It seems like we'll need more than a single boolean to keep track of different IPs.

Reply:
Sure.. just annoying having it spam. Maybe make a list of previously filtered ips and check against that